### PR TITLE
chore(shared-data): adds liquid-class Ethanol-80 and Glycerol-50

### PR DESCRIPTION
**Pull request by andySigler**
# Overview

JSON definitions for new liquid classes.

### CODE REVIEW

File: shared-data/liquid-class/definitions/1/glycerol_50.json [hunk 1]
```diff
@@ -0,0 +1,2322 @@
...
+          "tiprack": "opentrons_flex_96_tiprack_50ul",
+          "aspirate": {
+            "submerge": {
+              "positionReference": "well-top",
+              "offset": {
+                "x": 0,
+                "y": 0,
+                "z": 2
+              },
+              "speed": 4,
```
Comment (ddcc4) [inline shared-data/liquid-class/definitions/1/glycerol_50.json:20]: I'm just amused at how much slower we have to go for glycerol (compared to 100 mm/s for water).

File: shared-data/liquid-class/definitions/1/glycerol_50.json [hunk 2]
```diff
@@ -0,0 +1,2458 @@
...
+                }
+              }
+            },
+            "positionReference": "well-top",
+            "offset": {
+              "x": 0,
+              "y": 0,
+              "z": 2
+            },
+            "flowRateByVolume": [
```
Comment (jbleon95) [inline shared-data/liquid-class/definitions/1/glycerol_50.json:1992]: These values don't seem correct being negative for a flowrate, and are causing the JS schema validation tests to fail

Comment (ddcc4) [reply shared-data/liquid-class/definitions/1/glycerol_50.json:1992]: > These values don't seem correct being negative for a flowrate, and are causing the JS schema validation tests to fail

@jbleon95 Are you referring to the negative values in `correctionByVolume`? It seems like it should be legitimate to have a small negative number as the correction.

Comment (andySigler) [reply shared-data/liquid-class/definitions/1/glycerol_50.json:1992]: this is a mistake, fixing now

File: shared-data/liquid-class/definitions/1/glycerol_50.json (hunk 1 above)
Comment (andySigler) [reply shared-data/liquid-class/definitions/1/glycerol_50.json:20]: (FYI) liquid adheres to plastic, so when the tip retracts out from the liquid there will always be some amount of liquid stuck to the outside of the tip (hopefully small enough it instantly evaporates). With glycerol, it is so adhesive to the plastic that the faster the tip exits, the more liquid remains stuck on the tip. Moving slowly gives the liquid more time for cohesion + gravity to pull it "down" off the tip.

**Review by sanni-t — APPROVED** [review]
LGTM! 👍

**Pull request merged**